front/components/assistant/AssistantDropdownMenu.tsx

```diff
@@ -108,113 +110,92 @@ export function AssistantDropdownMenu({
-                icon={ChatBubbleBottomCenterTextIcon}
-                onClick={(e) => {
-                  e.stopPropagation();
-                  close();
-                }}
-              />
-              {isMoreInfoVisible ? (
-                <DropdownMenu.Item
-                  label="More info"
-                  onClick={() =>
-                    setQueryParam(
-                      router,
-                      "assistantDetails",
-                      agentConfiguration.sId
-                    )
-                  }
-                  icon={EyeIcon}
-                />
-              ) : (
-                <DropdownMenu.Item
-                  label={`Copy assistant ID`}
+      <NewDropdownMenu>
+        <NewDropdownMenuTrigger asChild>
+          {dropdownButton}
+        </NewDropdownMenuTrigger>
+        <NewDropdownMenuContent>
+          <NewDropdownMenuLabel>{agentConfiguration.name}</NewDropdownMenuLabel>
+          <NewDropdownMenuItem
+            label="Start new conversation"
+            icon={ChatBubbleBottomCenterTextIcon}
+            onClick={() =>
+              router.push(
+                `/w/${owner.sId}/assistant/new?assistant=${agentConfiguration.sId}`
+              )
+            }
+          />
+          {isMoreInfoVisible ? (
+            <NewDropdownMenuItem
+              label="More info"
+              icon={EyeIcon}
+              onClick={(e) => {
+                e.stopPropagation();
+                setQueryParam(
+                  router,
+                  "assistantDetails",
+                  agentConfiguration.sId
+                );
+              }}
+            />
+          ) : (
+            <NewDropdownMenuItem
+              label="Copy assistant ID"
+              icon={ClipboardIcon}
+              onClick={async (e) => {
+                e.stopPropagation();
+                await navigator.clipboard.writeText(agentConfiguration.sId);
+              }}
+            />
+          )}
+          {showAddRemoveToFavorite && (
+            <NewDropdownMenuItem
+              label={isFavorite ? "Remove from favorites" : "Add to favorites"}
+              icon={isFavorite ? StarStrokeIcon : StarIcon}
+              disabled={isUpdatingFavorites}
+              onClick={async (e) => {
+                e.stopPropagation();
+                await updateFavorite(!isFavorite);
+              }}
+            />
+          )}
+          {!isGlobalAgent && (
+            <>
+              <NewDropdownMenuSeparator />
+              <NewDropdownMenuLabel>Edition</NewDropdownMenuLabel>
+              {(isBuilder(owner) || !isAgentWorkspace) && (
+                <NewDropdownMenuItem
+                  label="Edit"
+                  icon={PencilSquareIcon}
                   onClick={async (e) => {
                     e.stopPropagation();
-                    await navigator.clipboard.writeText(agentConfiguration.sId);
-                    close();
+                    await router.push(
```

Comment:
Can we use the `<Link>` here? Ideally the `NewDropdownMenuItem` should accept a link or at least wrap it in ` <Link>`.